entity/Junde Yhi

```diff
@@ -0,0 +1,18 @@
+NAME="Junde Yhi"
+CONTACT=(
+	"EMAIL:lmy441900@live.com"
+	"TELEGRAM:@lmy441900"
+	"MASTODON:@lmy441900@sn.angry.im"
+	"GITHUB:lmy441900"
+)
+BABEL=(
+	"zh-N"
+	"zh-hans-N"
+	"zh-hant-2"
+  "en-3"
```

Comment:
plz use tab

Reply:
Fixed in 3f6d08a, thank you for pointing out this silly mistake.